css/style.css

```diff
@@ -465,3 +556,151 @@ footer {
   transition-duration: 0.7s;
   transform: scale(1.05);
 }
+
+.my_email {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+@media screen and (min-width: 930px) {
```

Comment:
- [ ] Since it is a requirement for this project, your desktop breakpoint value for `min-width` should be in the range `992px - 1100px` Please kindly fix that. See screenshot of the requirements below

![range](https://user-images.githubusercontent.com/62849872/150656890-9b48c7e3-9b5f-421a-84d3-32c1372d00b9.png)